LocalSettings.php

```diff
@@ -4256,8 +4258,118 @@
 	'wmgWikibaseRepoPropertyNamespaceID' => [
 		'default' => 862
 	],
+	'wmgWikibaseAllowLocalShortDesc' => [
+		'default' => false,
+		'gratispaideiawiki' => true,
+	],
+	'wmgWikibaseForceLocalShortDesc' => [
+		'default' => false,
+		'gratispaideiawiki' => true,
+	],
+	'wmgWikibaseEnableData' => [
```

Comment:
Again, why is it necessary to set the default here unless 1) the setting is in ManageWiki 2) you wish to set something different for your wiki?

Reply:
@Reception123 some of the configs are different for my wiki?

Reply:
Given in WikiBase.php we just set $wg = $wmg, why not just set the wg version here?

Reply:
Tbh, I think the wg versions should better be set in our Wikibase.php file, tbqh